src/content/blog/2025-03-25-tus-node-server-v2.0.0.md

```diff
@@ -0,0 +1,185 @@
+---
+title:
+  'tus Node.js server 2.0.0: integrate in all meta-frameworks and JS runtimes'
+author: murderlon
+date: 2025-03-25
+---
+
+The Node.js tus server can now run in all meta frameworks and Node.js compatible
+runtimes. Deprecated options have been removed, and the server is ESM-only. The
+packages now require Node.js >=20.19.0, the release that backported
+`require(esm)`, allowing us to output ESM and import ESM packages without
+affecting consumers on CommonJS. But that’s not all, let’s dive in.
+
+## What is tus?
+
+tus is an HTTP-based protocol for resumable file uploads. Resumable means that
+an upload can be picked up again at any moment, after it was interrupted for
+whatever reason, without the need to re-upload the previous data. An
+interruption may happen on purpose, if the user wants to pause, or by accident
+in case of a network issue or server outage (or your cat deciding to take a nap
+on the keyboard).
+
+The developers behind tus are currently working together with the influential
+HTTP working group inside the Internet Engineering Task Force (IETF) – the
+internet standards-setting organization that defines the network protocols
+powering the entire internet – to
+[make resumable uploads an official standard](https://tus.io/blog/2023/08/09/resumable-uploads-ietf)
+across the web.
+
+[tus-node-server][] is an official implementation of the tus resumable upload
+protocol. It is capable of accepting uploads of all sorts and sizes, and storing
```

Comment:
The previous section talked about the IETF involvement. Would it be fair to mention here that tus-node-server does not yet support the IETF draft?